czech.h

```diff
@@ -1,4 +1,52 @@
-#define SOUBOR FILE
-#define když if
-#define plav float
-#define číslo int
+#define SOUBOR      FILE
+
+#define nehybný     static
+#define brusle      inline
+#define měňavec     volatile
+#define autobus     auto
+#define venkovní    extern
+
+#define velikost    sizeof
+#define novytyp     typedef
+
+#define běž         goto
+#define pokud       if
+#define takne       else
+#define dělej       do
+#define dokud       while
+#define provse      for
+#define rozchoďit   continue
+#define rozhodit    break
+#define vrať        return
+#define přepínač    switch
+#define případ      case
+#define výchozí     default
+
+#define nabeton     true
+#define nepravda    false
+
+#define struktura   struct
+#define výčet       enum
+#define CCCP        union
+
+#define rozhodovák  bool
+#define neměnné     const
+#define podepsaný   signed
+#define nepodepsaný unsigned
+#define nic         void
+#define cvalík      double
+#define plavec      float
+#define říman       int
```

Comment:
co prosim

Reply:
?

Reply:
možná #define tuplák double
je lepší

Reply:
Jde mi o ten int

Reply:
tak římani používali celá čísla

Reply:
Jakože spíš bych se držel špatného překladu než referencí na docela unrelated věci

Reply:
tak to asi necháme třeba na #define čís int